docs/docs/user/sessions/troubleshooting/index.md

```diff
@@ -50,19 +50,40 @@
     If you have no success, please reach out your administrator. Administrators
     can see the logs of read-only sessions.
 
-??? question "The TeamForCapella password doesn't work"
+<!-- prettier-ignore -->
+??? question "The TeamForCapella password doesn't work <a href="#t4c-password-doesnt-work">:material-link:</a>"
+
+    <span id="t4c-password-doesnt-work" />
 
     Since we have released version 2, users have to use sessions passwords. More
-    information can be found [here](../tools/capella/teamforcapella/connect/index.md).
+    information can be found [here](../../tools/capella/teamforcapella/connect/index.md).
     If you have been added to a project recently while having a running session,
     please recreate your session to get a valid session password for the models
     of the required project.
 
 <!-- prettier-ignore -->
-??? question "I can't use the `Control` key for the secondary/right-click on macOS"
+??? question "I can't use the `Control` key for the secondary/right-click on macOS <a href="#macos-control-key">:material-link:</a>"
+
+    <span id="macos-control-key"/>
 
     Please use an external mouse or use the trackpad gesture (click with two
     fingers on the trackpad).
 
     The sessions run on Linux and the `Control` key can't be mapped properly
     via the remote desktop protocol.
+
+    Alterantively, you can use the Xpra connection method instead of Guacamole.
+
+<!-- prettier-ignore -->
+??? question "My Xpra window doesn't react anymore <a href="#xpra-reaction">:material-link:</a>"
+
+    <span id="xpra-reaction"/>
+
+    If the Xpra window doesn't react anymore, a dialog in the background might block the main window.
+
+    To identify a potential dialog, double-click on the title-bar of the Xpra window. Alternatively, you can minimize it:
+    ![Minimize Xpra window](./minimize-xpra.png)
+
+    Then, drag the window around to see if there are any other popups. If you find a popup, close it.
+
+    Another option is to refresh the browser tab. It will bring the popup to the front, but takes a bit longer.
```

Comment:
```suggestion
    Another option is to refresh the browser tab. It will bring the hidden modal dialog into the foreground. This step might take a bit longer.
```